components/console/helpers/progressindicator.rst

```diff
@@ -94,22 +97,23 @@ The progress indicator will now look like this:
      ⠛ Processing...
      ⠹ Processing...
      ⢸ Processing...
+     ✔ Finished
 
-Once the progress indicator is finished, it keeps the latest indicator value by
-default. You can replace it with your own::
+Once the progress indicator is finished, it uses the finishedIndicator value (which defaults to ✔). You can replace it with your own::
 
-    $progressIndicator->finish('Finished', '✅');
-
-Then the progress indicator will render like this:
-
-.. code-block:: text
+    $progressIndicator = new ProgressIndicator($output, 'verbose', 100, null, '🎉');
```

Comment:
I think we should include the "Processing ... Finished" output example here too so readers can see that 🎉 is the new finished indicator.

Reply:
Done while merging.